src/modules/0.0.10/aws_acm_import/index.ts

```diff
@@ -1,10 +1,28 @@
-import { ImportCertificateCommandInput } from '@aws-sdk/client-acm';
-import { AWS, } from '../../../services/gateways/aws_2';
+import { ImportCertificateCommandInput, paginateListCertificates, } from '@aws-sdk/client-acm'
+import { AWS, paginateBuilder, } from '../../../services/aws_macros'
 import { Context, Crud2, Mapper2, Module2, } from '../../interfaces'
-import { AwsAcmListModule } from '../aws_acm_list';
-import { CertificateImport } from './entity';
-import * as metadata from './module.json';
+import { AwsAcmListModule } from '../aws_acm_list'
+import { CertificateImport } from './entity'
+import * as metadata from './module.json'
 
+const getCertificatesSummary = paginateBuilder(paginateListCertificates, 'CertificateSummaryList');
+
+// TODO: Can I macro this somehow?
+async function importCertificate(client: any, input: ImportCertificateCommandInput) {
+  const res = await client.importCertificate(input);
+  const arn = res.CertificateArn ?? '';
+  let certificates: string[] = [];
+  let i = 0;
+   // Wait for ~1min until imported cert is available
+  do {
+    const start = Date.now();
+    await new Promise(r => setTimeout(r, 2000));
+    while (Date.now() - start < 2000); // Sleep for 2s
```

Comment:
oh you re copying here the fixed hot-loops issue

Reply:
:facepalm: